examples/list_disk_snapshots.py

```diff
@@ -64,8 +64,12 @@ def parse_args():
     # Create mapping from snapshot parent to snapshot, used to reconstruct the
     # snapshot chain.
     snapshots = {}
-    for s in disk_snapshots_service.list(include_active=True):
+
+    # Snapshots belonging to other disks will be added here
+    other_snapshots = []
```

Comment:
This should be dict, mapping snapshot id to snapshot info.

Reply:
Nice! Will rewrite it this way